src/index.html

```diff
@@ -9,6 +16,68 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a class="Logo" href="#home">
```

Comment:
```suggestion
      <a class="logo" href="#home">
```
No need to capitalize the class name